docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1620,6 +1620,41 @@ its value will be replaced with the provided one.
 }
 --------------------------------------------------
 
+[[size-processor]]
+=== Size Processor
+Sets the size of the current document as it will be rendered as a JSON document.
+
+[[size-options]]
+.Siz Options
+[options="header"]
+|======
+| Name                  | Required | Default      | Description
+| `target`              | no       | `_meta.size` | The target field the size will be stored in
```

Comment:
I would avoid using a field that starts with an underscore since we use it for internal mappers (eg. _id). Should we make this parameter required?